src/firefly/js/templates/lightcurve/LcPhaseTable.js

```diff
@@ -227,7 +229,7 @@ function repeatDataCycle(phaseTable) {
     slice(tableData.data, 0).forEach((d) => {
         var newRow = slice(d);
 
-        newRow[fIdx] = `${parseFloat(d[fIdx]) + 1}`;
+        newRow[fIdx] = `${ (parseFloat(d[fIdx]) + 1).toFixed(DEC_PHASE)}`;
         tableData.data.push(newRow);
```

Comment:
Now we have to understand why this is not failing in OPS (???) to make sure we are not forgetting to fix something else behind that might trigger somewhere else.

Reply:
That was exactly my question which I asked in the ticket IRSA-3112.   It was very strange.
I played in the console:
0.257+1
1.2570000000000001
0.128+1
1.1280000000000001

I searched online and learned that there are precision problems in javascript.  Does OPS run the same version if javascript/react?  Maybe Loi @loitly has more insight about this issue.

Reply:
I can't explain why it's different in OPS and DEV.  Unless they have different data/numbers and those data happens to fall into the category where there's no `bad` round off at the end.